zrml/market-commons/src/lib.rs

```diff
@@ -152,10 +158,7 @@ mod pallet {
             <Markets<T>>::iter()
         }
 
-        fn market(
-            market_id: &Self::MarketId,
-        ) -> Result<Market<Self::AccountId, Self::BlockNumber, Self::Moment>, DispatchError>
-        {
+        fn market(market_id: &Self::MarketId) -> Result<MarketOf<T>, DispatchError> {
```

Comment:
You can use this cool approach [here](https://github.com/zeitgeistpm/zeitgeist/blob/d6a3b8a5e1d056127c51f00f2f02be3409ebe742/zrml/market-commons/src/lib.rs#L181), [here](https://github.com/zeitgeistpm/zeitgeist/blob/d6a3b8a5e1d056127c51f00f2f02be3409ebe742/zrml/market-commons/src/lib.rs#L168), [here](https://github.com/zeitgeistpm/zeitgeist/blob/d6a3b8a5e1d056127c51f00f2f02be3409ebe742/zrml/market-commons/src/lib.rs#L238) and [here](https://github.com/zeitgeistpm/zeitgeist/blob/d6a3b8a5e1d056127c51f00f2f02be3409ebe742/zrml/market-commons/src/lib.rs#L156).